.gitpod.yml

```diff
@@ -0,0 +1 @@
+mainConfiguration: https://github.com/mattermost/mattermost-gitpod-config
```

Comment:
0/5, but unless we use this we can just delete.

Reply:
This stuff is just part of the starter-template.  Deleting.